### PR TITLE
Refactor Tenant Management Guide

### PR DESCRIPTION
**Pull request by nilminiwso2**
Note that the topics moved under `User Management` will be improved by the following PR [1].

[1] https://github.com/wso2/docs-is/pull/3186

### CODE REVIEW

File: en/docs/apis/keystore-rest-api.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ template: templates/swagger.html
 
 ??? Note "Click for instructions"
     Follow the steps given below to try out the REST APIs with your local instance of WSO2 Identity Server. 
-    To try some APIs, a tenant needs to be created with the domain name, 'wso2.com'. See [Add a new tenant]({{base_path}}/guides/tenants/add-new-tenants) for more information.
+    To try some APIs, a tenant needs to be created with the domain name, 'wso2.com'. See [Add a new tenant]({{base_path}}/guides/tenants/tenant-mgt) for more information.
```
Comment (nilminiwso2) [inline en/docs/apis/keystore-rest-api.md:9]:
```suggestion
    To try some APIs, a tenant needs to be created with the domain name 'wso2.com'. See [Add a new tenant]({{base_path}}/guides/tenants/tenant-mgt) for more information.
```

File: en/docs/deploy/configure-email-sending.md [hunk 2]
```diff
@@ -46,5 +46,116 @@ This document explains the steps to configure WSO2 Identity Server to send email
...
+
+3.  Execute the following curl command for creating a resource named `EmailPublisher`. 
+
+    **Sample Request**
+    ``` java 
+    curl -X POST "https://localhost:9443/t/{tenant-domain}/api/identity/config-mgt/v1.0/resource/Publisher" -H "accept: 
+    application/json" -H "Content-Type: application/json" -H 'Authorization: Basic YWRtaW46YWRtaW4=' -d "{ \"name\": \"EmailPublisher\", \"attributes\": [ { \"key\": \"email\", \"value\": \"string\" } ]}"
+    ```
+	
+4.  Execute the following curl command for creating a file named `EmailPublisher`.
```
Comment (himeshsiriwardana) [inline en/docs/deploy/configure-email-sending.md:71]: Can we differentiate between point 3 and point 4? They have the same instructions.

File: en/docs/deploy/configure-email-sending.md [hunk 3]
```diff
@@ -46,5 +46,116 @@ This document explains the steps to configure WSO2 Identity Server to send email
...
+        This `EmailPublisher.xml` file will be used as the tenant's email publisher file. Configure the tenant-wise email configurations in the `EmailPublisher.xml` file.
+
+    **Sample Request**
+    ``` java 
+    curl -X POST "https://localhost:9443/t/{tenant-domain}/api/identity/config-mgt/v1
+    .0/resource/Publisher/EmailPublisher/file" -H "accept: application/json" -H 
+    "Content-Type: multipart/form-data" -H 'Authorization: Basic YWRtaW46YWRtaW4=' -F "resourceFile=@EmailPublisher.xml;type=text/xml" -F "fileName=EmailPublisher"
+    ```
+    
+5.  Open the `EmailPublisher.xml` file and configurable the parameters given below.
```
Comment (himeshsiriwardana) [inline en/docs/deploy/configure-email-sending.md:83]:
```suggestion
5.  Open the `EmailPublisher.xml` file and configure the parameters given below.
```

File: en/docs/deploy/configure-email-sending.md [hunk 4]
```diff
@@ -46,5 +46,116 @@ This document explains the steps to configure WSO2 Identity Server to send email
...
+
+    !!! note
+        -   Only one `EmailPublisher.xml` file with the name `EmailPublisher` should be added for a tenant.
```
Comment (himeshsiriwardana) [inline en/docs/deploy/configure-email-sending.md:86]:
```suggestion
        -   Only one `EmailPublisher.xml` file with the name `EmailPublisher` should be added to a tenant.
```

File: en/docs/guides/tenants/tenant-mgt.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
+# Tenant Management
+
+By default, WSO2 Identity Server is configured with a single tenant named `carbon.super`. If you have the required permissions, you can configure multiple tenant domains to allow multiple user bases to log in and use a single server/cluster at the same time in anisolated manner.
```
Comment (himeshsiriwardana) [inline en/docs/guides/tenants/tenant-mgt.md:3]:
```suggestion
By default, WSO2 Identity Server is configured with a single tenant named `carbon.super`. If you have the required permissions, you can configure more tenant domains and maintain multiple isolated user bases on a single server/cluster at the same time.
```

File: en/docs/guides/tenants/tenant-mgt.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+    | **Admin Password**               | The password used to sign in using the admin username specified.                                                                                                                                                                                                                                   |
+    | **Admin Password (Repeat)**      | Repeat the password to confirm.                                                                                                                                                                                                                                                                   |
+    | **Email**                        | The email address of the admin.                                                                                                                                                                                                                                                                   |
+
+Once a new tenant is created, the tenant details appears in the **Tenants List** as shown below.
```
Comment (himeshsiriwardana) [inline en/docs/guides/tenants/tenant-mgt.md:33]:
```suggestion
Once a new tenant is created, the tenant details appear in the **Tenants List** as shown below.
```

**Review by himeshsiriwardana — APPROVED** [review]

File: en/docs/guides/tenants/tenant-mgt.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+    | **Select Usage Plan for Tenant** | The usage plan defines limitations (such as the number of users, bandwidth etc.) for the tenant.                                                                                                                                                                                                  |
+    | **First Name** / **Last Name**   | The name of the tenant admin.                                                                                                                                                                                                                                                                     |
+    | **Admin Username**               | The sign-in username of the tenant admin. The username always ends with the domain name (e.g., admin@abc.com ) .                                                                                                                                                                                     |
```
Comment (divyaamunugama) [inline en/docs/guides/tenants/tenant-mgt.md:28]:
```suggestion
    | **Admin Username**               | The sign-in username of the tenant admin. The username always ends with the domain name (e.g., admin@abc.com ).                                                                                                                                                                                     |
```

File: en/docs/guides/tenants/tenant-mgt.md [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+    |----------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+    | **Domain**                       | The domain name (e.g., abc.com), which is used as a unique identifier for your domain. You can use it to sign in to the Management Console to be redirected to your specific tenant. The domain is also used in URLs to distinguish one tenant from another. |
+    | **Select Usage Plan for Tenant** | The usage plan defines limitations (such as the number of users, bandwidth etc.) for the tenant.                                                                                                                                                                                                  |
```
Comment (divyaamunugama) [inline en/docs/guides/tenants/tenant-mgt.md:26]:
```suggestion
    | **Select Usage Plan for Tenant** | The usage plan defines limitations (such as the number of users, bandwidth, etc.) for the tenant.                                                                                                                                                                                                  |
```

**Review by divyaamunugama — APPROVED** [review]

File: en/docs/deploy/configure-email-sending.md (hunk 2 above)
Comment (nilminiwso2) [reply en/docs/deploy/configure-email-sending.md:71]: Noted. Need to test this.
Shall we fix these during the testing effort tracked by the excel? Only did the refactoring here.

**Pull request merged**